src/main/java/org/ow2/proactive/procci/model/utils/ConvertUtils.java

```diff
@@ -40,6 +40,17 @@
         }
     }
 
+    public static Optional<String> getStringFromObject(Optional<Object> stringObject) throws SyntaxException {
```

Comment:
rename it to convertToString, but normally , use simply object.toString() does the job, a dedicated method for this might not be necessary